webpack.config.js

```diff
@@ -49,9 +49,7 @@ module.exports = {
     },
     resolve: {
         extensions: [".js", ".jsx"],
-        alias: {
-            esquery: path.resolve(__dirname, "src/js/demo/node_modules/esquery/dist/esquery.js")
-        }
+        mainFields: ["browser", "main", "module"]
```

Comment:
Rather than modifying this directly, could we set `target: "web"` instead? It looks like this is the default value used when the target is set.

Reply:
By the [resolve.mainFields](https://webpack.js.org/configuration/resolve/#resolvemainfields) docs I think it would be `["browser", "module", "main"]`?

And that `"module"` before `"main"` is causing problems with `esquery` since it results in a different API:

 `{ default: { parse, match, matches, query } }`

Reply:
Ah, I see. This makes sense to me :+1: